node/src/Channel.ts

```diff
@@ -275,6 +288,80 @@ export class Channel extends EnhancedEventEmitter
 		});
 	}
 
+	async requestBinary(
+		method: Method,
+		bodyType?: RequestBody,
+		bodyOffset?: number,
+		handlerId?: string): Promise<Response>
+	{
+		this.#nextId < 4294967295 ? ++this.#nextId : (this.#nextId = 1);
+
+		const id = this.#nextId;
+
+		logger.error('request() [method:%s, id:%s]', id);
+
+		if (this.#closed)
+			throw new InvalidStateError('Channel closed');
+
+		const handlerIdOffset = this.#bufferBuilder.createString(handlerId);
+
+		let requestOffset: number;
+
+		if (bodyType && bodyOffset)
+		{
+			requestOffset = Request.createRequest(
+				this.#bufferBuilder, id, method, handlerIdOffset, bodyType, bodyOffset);
+		}
+		else
+		{
+			requestOffset = Request.createRequest(
+				this.#bufferBuilder, id, method, handlerIdOffset, RequestBody.NONE, 0);
+		}
+
+		this.#bufferBuilder.finish(requestOffset);
+
+		const buffer = this.#bufferBuilder.asUint8Array();
+
+		if (buffer.byteLength > MESSAGE_MAX_LEN)
+			throw new Error('Channel request too big');
+
+		// This may throw if closed or remote side ended.
+		this.#producerSocket.write(
+			Buffer.from(Uint32Array.of(buffer.byteLength).buffer));
+		// Set buffer enconding to 'binary.'
+		this.#producerSocket.write(buffer, 'binary');
+
+		return new Promise((pResolve, pReject) =>
+		{
+			const sent: Sent =
+			{
+				id      : id,
+				method  : '',
```

Comment:
?

Reply:
yep, missing, typo. BTW now methods are not strings, but enums.